Makefile

```diff
@@ -247,21 +247,19 @@ swagger-docs: ## preview the API documentation
 
 .PHONY: buildx
 ifeq ($(BUILDX), bundles/buildx)
-buildx: bundles/buildx # build buildx cli tool
-else
-buildx:
+buildx: bundles/buildx ## build buildx cli tool
 endif
 
-bundles/buildx: BUILDX_DOCKERFILE ?= Dockerfile.buildx
-bundles/buildx: BUILDX_COMMIT ?= v0.3.0
+# This intentionally is not using the `--output` flag from the docker CLI, which
+# is a buildkit option. The idea here being that if buildx is being used, it's
+# because buildkit is not supported natively
 bundles/buildx: bundles ## build buildx CLI tool
-	# This intetionally is not using the `--output` flag from the docker CLI which is a buildkit option
-	# The idea here being that if buildx is being used, it's because buildkit is not supported natively
-	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$(BUILDX_COMMIT)" \
+	docker build -f $${BUILDX_DOCKERFILE:-Dockerfile.buildx} -t "moby-buildx:$${BUILDX_COMMIT:-latest}" \
 		--build-arg BUILDX_COMMIT \
 		--build-arg BUILDX_REPO \
 		--build-arg GOOS=$$(if [ -n "$(GOOS)" ]; then echo $(GOOS); else go env GOHOSTOS || uname | awk '{print tolower($$0)}' || true; fi) \
 		--build-arg GOARCH=$$(if [ -n "$(GOARCH)" ]; then echo $(GOARCH); else go env GOHOSTARCH || true; fi) \
```

Comment:
oh! `$(GOOS)` and `$(GOARCH)` are Make variables as well 🤔. I wonder why it worked before though

Reply:
Perhaps I need to borrow this snippet from containerd; https://github.com/containerd/containerd/blob/036db34f37617d6a02f07b04fc2d35a91732c6e3/Makefile#L27-L52

Or set it in the Jenkinsfile

Reply:
Ah! LOL.. then it worked, but didn't work because now we _actually_ used `v0.3.0`, which doesn't support s390x/ppc64le; The `copy` image use in v0.3.0 does not have s390x/ppc64le;

```
#3 docker-image://docker.io/tonistiigi/copy:v0.1.3@sha256:e57a3b4d6240f55ba...
#3 digest: sha256:150c626a5355601b2c71794d6e4a9af463e674e78c8f6cd2b0839485d62d8c5f
#3 name: "docker-image://docker.io/tonistiigi/copy:v0.1.3@sha256:e57a3b4d6240f55bac26b655d2cfb751f8b9412d6f7bb1f787e946391fb4b21b"
#3 started: 2019-10-14 16:57:58.920184883 +0000 UTC
#3 completed: 2019-10-14 16:57:58.920265067 +0000 UTC
#3 duration: 80.184
#3 started: 2019-10-14 16:57:58.920324378 +0000 UTC
#3 completed: 2019-10-14 16:57:58.922283274 +0000 UTC
#3 duration: 1.958896ms
#3 error: "no match for current platform linux/s390x in manifest sha256:e57a3b4d6240f55bac26b655d2cfb751f8b9412d6f7bb1f787e946391fb4b21b: not found"
#3 resolve docker.io/tonistiigi/copy:v0.1.3@sha256:e57a3b4d6240f55bac26b655d2cfb751f8b9412d6f7bb1f787e946391fb4b21b done
```

Reply:
See my other comment; it's because this PR adds a `COPY`, and therefore `docker.io/tonistiigi/copy:v0.1.3` has to be pulled, which is only available for `amd64` and `armv7`